### ISSUE TITLE
Log errors when offline

### ISSUE THREAD

**Issue by dabeeeenster** [ellie/atuin#516]
I'm getting errors when the sync server is not reachable that are polluting my terminal - is this just me? `error sending request for url` and some additional items?

Thanks for the project - use it multiple times a day!

**Comment by ellie** [comment]
Happy to hear you like Atuin!

Hmm those errors shouldn't be making their way to your terminal. 

Just a couple of questions 

What shell do you use?

How have you setup the atuin integration?

**Comment by dabeeeenster** [comment]
Hi,

Using fish shell on MacOS. Getting this sort of thing:

<img width="1436" alt="image" src="https://user-images.githubusercontent.com/173290/187851978-a83b8df4-6df5-42a3-b706-4d6206557b64.png">

Integrating like this in `fish.config`:

```
    # Try https://github.com/ellie/atuin
    # dont bind to up arrow
    set ATUIN_NOBIND true
    atuin init fish | source
    # But bind to CTRL+R
    bind \cr _atuin_search
```

**Comment by conradludgate** [comment]
Hmm, I'm not to familiar with fish, but it does look like we are missing the `/dev/null` redirect that we use for bash/zsh. Maybe that's the issue

**Comment by conradludgate** [comment]
I can confirm. We are currently using `&; disown` to spawn the end command in the background, but my testing shows it still outputs the stdout/err. We just need to add `&>/dev/null` and it should be good

**Comment by ellie** [comment]
Resolved by #518
